js/plugins/code/link-element.js

```
@@ -0,0 +1,521 @@
+            case 'Delete':
+                event.preventDefault();
+                wisk.editor.deleteBlock(this.id);
+                break;
+            case 'Enter':
+                event.preventDefault();
+                wisk.editor.createNewBlock(this.id, 'text-element', { textContent: '' }, { x: 0 });
+                break;
+            case 'ArrowUp':
+            case 'ArrowLeft':
+                event.preventDefault();
+                const prevElement = wisk.editor.prevElement(this.id);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)